File: benchmarking/GenSSI/Goodwin-oscillator.m
function model = SMTH()
syms x2 x4 x1 x3
syms b alpha c gama delta sigma beta
syms x20 x40 x10 x30
model.sym.p = [b; alpha; c; gama; delta; sigma; beta; x20; x40; x10; x30];
model.sym.x = [x2; x4; x1; x3];
model.sym.g = [];
model.sym.x0 = [x20; x40; x10; x30];
model.sym.xdot = [alpha*x1 - beta*x2
(gama*sigma*x2*x4 - delta*sigma*x3*x4) / (x3)
(-b*c*x1 - b*x1*x4 + 1) / (c + x4)
gama*x2 - delta*x3];
model.sym.y = [x1];
end

File: benchmarking/IdentifiableFunctions/results/Goodwin oscillator/Goodwin oscillator.jl
# Goodwin oscillator
#! format: off
using StructuralIdentifiability

system = @ODEmodel(
	x1'(t) = (-b*c*x1(t) - b*x1(t)*x4(t) + 1)//(c + x4(t)),
	x2'(t) = alpha*x1(t) - beta*x2(t),
	x3'(t) = gama*x2(t) - delta*x3(t),
	x4'(t) = (gama*sigma*x2(t)*x4(t) - delta*sigma*x3(t)*x4(t))//x3(t),
	y(t) = x1(t)
)
